docker/python-verify/conda.Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} as base_image
+LABEL maintainer="Apache NiFi <dev@nifi.apache.org>"
+
+ENV USER minificpp
+ENV CONDA_HOME /opt/conda
+ENV LD_LIBRARY_PATH /opt/conda/lib
+ENV MINIFI_BASE_DIR /opt/minifi
+ENV MINIFI_HOME ${MINIFI_BASE_DIR}/minifi-current
+
+USER root
+
+RUN wget https://repo.anaconda.com/archive/Anaconda3-2022.10-Linux-x86_64.sh -P /tmp
```

Comment:
is there some way to verify the hash of the downloaded script?

Reply:
Good idea, I've added sha256 verification in https://github.com/apache/nifi-minifi-cpp/pull/1504/commits/267bdc2e2f2ad4926d82f048cc1c2a21c34e5f1d